apps/docs/app/features/routes/ressurser/kom-i-gang/Component.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Heading,
+  Flex,
+  Image,
+  SimpleGrid,
+  Stack,
+  Text,
+} from "@vygruppen/spor-react";
+
+export default function Component() {
+  return (
+    <SimpleGrid columns={{ base: 1, md: 2 }} spacing={8}>
```

Comment:
Pirk: I resten av kodebasen så har vi brukt arrays til å spesifisere de forskjellige breakpoint-verdiene. Kanskje vi skal gjøre det her også, bare for å være konsekvente?

```suggestion
    <SimpleGrid columns={[1,2]} spacing={8}>
```